bridge/src/main/java/io/stargate/bridge/service/ValuesHelper.java

```diff
@@ -122,12 +123,12 @@ public static BoundStatement bindValues(Prepared prepared, Values values, ByteBu
 
   public static ResultSet processResult(Rows rows, QueryParameters parameters)
       throws StatusException {
-    return processResult(rows, parameters.getSkipMetadata(), null, null, null, null);
+    return processResult(rows, parameters.getSkipMetadata(), null, null, null, null, null);
```

Comment:
Maybe use `QueryOuterClass.ResumeMode.UNRECOGNIZED` instead of `null`? `QueryParameters#getResumeMode()` will no return `null` but will instead return `UNRECOGNIZED`.